app/common/components/game-progress/game-progress.js

```diff
@@ -35,6 +35,10 @@ function getMarkText(index, currentRoundIndex) {
 }
 
 function getProgressFactor(currentRoundIndex, totalRounds) {
+    if (currentRoundIndex < 0) {
```

Comment:
it's mapped to it. Just to be in sync with `currentRoundIndex` and `selectedRoundIndex`